### PR TITLE
🇺🇦 Support Ukraine

### PR DESCRIPTION
**Pull request by alexandrtovmach**
сc: @zeke @Trott @obensource @MaledongGit @joesepi 

Added banner to support Ukraine!

image

I'm Alex, and I'm software engineer from Ukraine. Right now, my peaceful country attacked by neighborhood, without reason. It's really hard to understand from news, what's actually happening here

https://twitter.com/ua_parliament/status/1498266783173824517?t=XgTAeDu5-VH60dYs46m9fg&s=19

Our brave Ukrainian army show their strength, and fighting with one of the biggest army in the world on the same level and better. But Putin side is attacking civilians from the sky. It's violence that not acceptable by any war conflict, and we should stop it.

I've never asked for help, but I should do it now. I don't like to mix political things with job, but right now, my family should cover in a shelter 3 times a day, because of bombing. We are community, we can do many things. Help with paying attention to closing the sky over Ukraine by NATO, the rest, we can do ourselves.

We did support for BLM, and won, right now Ukraine need your help.

### CODE REVIEW

**Review by joesepi — APPROVED** [review]
I haven't had a chance to fully review this from a technical standpoint but fully support the message.

In the future, we may want to abstract out any sort of additional messaging with more abstract CSS classes and such.

🇺🇦 ✊

**Comment by alexandrtovmach** [comment]
haven't enought time to do CSS updates, but yeah we can do that

UPD: done

**Comment by ronag** [comment]
#CloseTheSky is controversial IMO. Can we use a different message? #SupportUkraine or something?

File: locale/ar/site.json
Comment (jasnell) [inline locale/ar/site.json:171]: Given that it is possible (and more common) for individual tweets to be deleted, This link should likely go to the profile instead (https://twitter.com/ua_parliament).

**Comment by benjamingr** [comment]
Hey is there anything more concrete Node.js can do to help? This sort of initiative can end up being performative.

If there are any areas where project members can contribute I would much rather we do that as a project instead of put another banner up. I'm personally disappointed with the commitments the project made with the BlackLivesMatter banner and then failed to meet. I think that initiative was full of good intent but I don't think we delivered on those commitments.

What about any of the following instead to show support:
 - Doing virtual code+learn events for Ukranians
 - Doing "ask a developer" sessions for local communities or panels
 - Helping local nonprofits with technical issues during the crisis
 - Helping people stuck in war zones with logistical issues (writing to consulates of your nationality asking for help etc)
 - Doing a fundraiser workshop (where proceeds go to a relief organization like the Ukranian red cross)

If any of those make sense to you I think it would be good for the TSC to discuss this and send out a "call for volunteers" to all Node.js org members. I think this can also go alongside a blog-post explaining what Node.js is doing.

I would also strongly prefer the focus here on helping Ukranians stuck in this situation and _not_ the war itself. Node.js has a diverse and multinational contributor base and I would not want to alienate members based on picking a side in a conflict most of us are pretty ignorant of.

**Comment by jasnell** [comment]
Overall, I'm definitely +1 on this. Like @ronag, I'd prefer something like `#SupportUkraine`.

File: locale/ar/site.json
Comment (alexandrtovmach) [reply locale/ar/site.json:171]: I believe this tweet can show what situation it is, and what kind of help we need. Adding just a link to profile will be boring and useful for anyone.

**Comment by benjamingr** [comment]
Also @alexandrtovmach stay strong and stay safe. I'm really sorry you have to go through this. I've spent time in bomb shelters before and it really sucks. I hope the conflict ends soon and your life goes back to normalcy. If there is any logistics I can help you with or you just wanna talk please reach out (benjamingr@gmail.com).

**Comment by alexandrtovmach** [comment]
@ronag @jasnell Changed message to `#SupportUkraine`

**Comment by alexandrtovmach** [comment]
@benjamingr For sure, community can help with:

> Helping local nonprofits with technical issues during the crisis
> Helping people stuck in war zones with logistical issues (writing to consulates of your nationality asking for help etc)
> Doing a fundraiser workshop (where proceeds go to a relief organization like the Ukranian red cross)

But I'm not sure how open source community can do that.
I'm also not asking to do anything with war, and only want to pay attention to the problem over the world, and push politics to resolve it by stoping violence. It's possible to do only in diplomatic way

**Comment by benjamingr** [comment]
@alexandrtovmach 

> For sure, community can help with:

I think a good next step to make that happen is to tag that issue with tsc-agenda and ask them to send a call for volunteers. If you know any local tech communities in your area you can contact them and see if you can reach non-profits working on relief.

As for a fundraiser workshop that's something the TSC would have to organize. I think "Node.js commits NNNNN$ to support Ukraine war relief" is a __much__ more powerful message than just a hashtag on the homepage.

**Comment by alexandrtovmach** [comment]
I've added tag, but I can't guarantee that I'll join the call with TSC, because of unstable situation. So for now, for the first step publishing hashtag to the website will be good.

**Comment by alexandrtovmach** [comment]
@benjamingr @ronag @jasnell would you approve?

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Qard — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
I'd like to have at least 2 approves from @nodejs/tsc members before merge it. Thanks everyone

**Review by ronag — APPROVED** [review]

**Review by tniessen — APPROVED** [review]
LGTM with no strong opinion about the exact link, and not sure about the exact subset of locales.

I agree with @benjamingr that actionable steps would be extremely valuable.

**Review by jasnell — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
Thanks for approve
Merging

**Comment by alexandrtovmach** [comment]
If we'll think that link is not works, we can fix it easily

**Comment by Trott** [comment]
No need to block on this comment, but it might be good to link (instead of to a Twitter account) to a relief organization that people can donate to in order to help Ukrainian refugees fleeing to neighboring countries. Then at least we're reminding/encouraging people to spend a little money to help out with the humanitarian issues. Perhaps https://donate.unhcr.org/int/en/ukraine-emergency?

**Comment by ronag** [comment]
I'm not sure what the rules are on this repo but wouldn't it be appropriate to wait at least 24 hours before we go live with this so that people have a chance to object?

**Comment by benjamingr** [comment]
I strongly agree with @ronag that there should be more eyes on this before this goes live so people have a chance to object.

More over I strongly prefer any of those options https://github.com/nodejs/nodejs.org/pull/4446#issuecomment-1054602348 to the banner after last time.

**Comment by alexandrtovmach** [comment]
ohh, sorry, reverting now for 24 hours

**Comment by alexandrtovmach** [comment]
is there a way to do revert with Github UI?

**Comment by Trott** [comment]
I guess we need to re-open this PR somehow or else open a new one?

**Pull request merged**
